amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/SecurityRequirementsParsers.scala

```diff
@@ -119,7 +129,7 @@ case class OasLikeSecurityRequirementParser(node: YNode,
   private def getScopes(schemeEntry: YMapEntry) =
     schemeEntry.value
       .as[Seq[YNode]]
-      .map(n => Scope(n).set(ScopeModel.Name, AmfScalar(n.as[String]), Annotations(n)))
+      .map(n => Scope(n).set(ScopeModel.Name, ScalarNode(n).text(), Annotations.inferred()))
```

Comment:
Lexical information which was previously present is lost

Reply:
Same here, lexical info for value.value should be present right?
We lost invalid value annotations (the one of the field) that now is maked as inferred